examples/meta/src/kernel/auc.sg

```diff
@@ -1,13 +1,13 @@
 File f_feats_train = csv_file("../../data/classifier_binary_2d_linear_features_test.dat")
 File f_labels_train = csv_file("../../data/classifier_binary_2d_linear_labels_test.dat")
 Features feats_train = features(f_feats_train)
-BinaryLabels labels_train(f_labels_train)
+Labels labels_train = labels(f_labels_train)
 
 Kernel subkernel = kernel("GaussianKernel", log_width=1.7)
 subkernel.init(feats_train, feats_train)
 
-AUCKernel auc(0, subkernel)
-auc.setup_auc_maximization(labels_train)
+Kernel auc = kernel("AUCKernel", subkernel=subkernel, labels=labels_train)
+auc.get_int("setup_auc_maximization")
```

Comment:
@karlnapf it is a bit awkward that it isn't `void`.. is it ok?

Reply:
Lol yeah that’s possible. But I think it is awkward indeed. Having the get compute something lazily is ok imo as the user wants to have something. But here the user wants to execute something so I think there should be a special method for that, hence “run” which indicates a void method